packages/contracts-bedrock/contracts/L1/L1StandardBridge.sol

```diff
@@ -91,14 +101,22 @@ contract L1StandardBridge is StandardBridge, Semver {
     );
 
     /**
-     * @custom:semver 1.1.0
+     * @notice Address of the balance claimer contract.
+     * @dev This contract is responsible for claiming the ERC20 balances of the bridge.
+     */
+    IBalanceClaimer public immutable BALANCE_CLAIMER;
+
```

Comment:
really big NIT, but these contracts never interact with `BALANCE_CLAIMER`, so storing it casted to an interface (and importing it) is not necessary. it could be stored as a plain `address` (you're actually casting it to address when comparing to `msg.sender`)

Reply:
yes actually did that in the previous pr because any interface function was used but requested to change https://github.com/defi-wonderland/wind-down/pull/1#discussion_r1801376048 to interface. I can change it again.